include/integratorxx/quadratures/gausscheby3.hpp

```diff
@@ -62,14 +62,14 @@ struct quadrature_traits<GaussChebyshev3<PointType, WeightType>> {
 
     weight_container weights(npts);
     point_container points(npts);
-    for (size_t i = 1; i <= npts; ++i) {
+    for (size_t i = 0; i < npts; ++i) {
       // The standard nodes and weights are given by
-      points[i] = std::cos(0.5 * (2*i - 1) * pi_ov_2n_p_1);
+      points[i] = std::pow(std::cos(0.5 * (2*(i+1) - 1) * pi_ov_2n_p_1),2);
```

Comment:
@susilehtola Would like a second pair of eyes here - based on the AS equations, this is the proper expression

Reply:
Yeah, this looks more correct